tests/integration/_experimental/object_detection/test_evaluator_multiclass_fixed.py

```diff
@@ -860,36 +813,41 @@ def assert_curves(
         assert sum(abs(a - b) for a, b in zip(curve.x, expectation.x)) < 1e-12
         assert len(curve.y) == len(expectation.y)
         assert sum(abs(a - b) for a, b in zip(curve.y, expectation.y)) < 1e-12
+        for extra_key in curve.extra.keys():
+            assert sum(abs(a - b) for a, b in zip(curve.extra[extra_key], expectation.extra[extra_key])) < 1e-12
 
 
 def assert_test_case_plots_equals_expected(
     plots: List[Plot],
     other_plots: List[Plot],
 ) -> None:
     assert len(plots) == len(other_plots)
-    # check curve plots
-    for plot, expected in zip(plots[:2], other_plots[:2]):
-        assert plot.title == expected.title
-        assert plot.x_label == expected.x_label
-        assert plot.y_label == expected.y_label
-        assert_curves(plot.curves, expected.curves)
-        assert plot.x_config == expected.x_config
-        assert plot.y_config == expected.y_config
+    # check one plot (curve.extra contains other plot)
```

Comment:
Any reason we don't verify plots[1], even if `curve.extra` for first plot contain similar info?